.ci/check_jenkins_job_status.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash +ex
+
+# Checks whether a specified Jenkins job succeeded in the last build.
+#
+# Expects $JENKINS_URL (url of Jenkins) to be set as environment variables
+#
+# Usage: check_jenkins_job_status.sh <check_job_name>
+#
+# Example: check_jenkins_job_status.sh  master
+
+# ANSI color codes
+RED="\e[31m"
+GREEN="\e[32m"
+BLUE="\e[34m"
+NC="\e[0m"
+
+CHECK_JOB_NAME=$1
+
+# Use Python to parse the JSON as it is usually installed on a machine
+JOB_STATUS=$(curl -s "$JENKINS_URL/job/$CHECK_JOB_NAME/lastBuild/api/json" | python -c "import sys, json; print json.load(sys.stdin)['result']")
```

Comment:
just hint: with installed jq it would be `| jq .result`

Reply:
Yeah, but without the `jq` installed, I would have to install `jq` :smile: